include/swift/Basic/OwnedString.h

```diff
@@ -44,10 +44,15 @@ class OwnedString {
   size_t Length;
   StringOwnership Ownership;
 
+  void release() {
+    if (Ownership == StringOwnership::Copied)
+      free(const_cast<char *>(Data));
```

Comment:
Quick drive by: Why are we freeing this. Why are we not using delete []?

On another note, I am just happy we are catching these leaks.